### ISSUE TITLE
UID, GID not applied while archiving webpage

### ISSUE THREAD

**Issue by foxisawesome** [go-shiori/shiori#1008]
## Data
- **Shiori version**: tag: latest
- **Database Engine**: Postgres 15-alpine
- **Operating system**: docker on centos steam 9
- **CLI/Web interface/Web Extension**: nano

## Describe the bug / actual behavior
I configured environment variables: UID and GID in docker compose yaml. And persist `/data` on local file system. However, while archiving web page, the ownership of the newly created web files on `/data` dir, is still `root`.

Is this is right way to do shiori root-less? My need is to make archived files belongs to UID:GID.

Thank you!

**Comment by fmartingr** [comment]
Hey, have you tried with the `--user` docker flag or the `user` docker compose attribute?

**Comment by foxisawesome** [comment]
Yes I tried this option. Using docker ‘user’ argument, shiori will not even work properly (it won’t initiate) using latest tag. Using env variable, it just does not behave as expected (create file as root) but program still works.On Nov 16, 2024, at 9:21 PM, Felipe Martin ***@***.***> wrote:﻿
Hey, have you tried with the --user docker flag or the user docker compose attribute?
